.github/release-drafter.yml

```
@@ -3,28 +3,28 @@ tag-template: 'v$RESOLVED_VERSION'
 categories:
   - title: '💥 Breaking Changes'
     labels:
-      - breaking
+      - 💥 breaking
   - title: '✨ Enhancements'
     labels:
       - feature
-      - enhancement
+      - ✨ enhancement
   - title: '🐛 Bug Fixes'
     labels:
       - fix
       - bugfix
-      - bug
+      - 🐛 bug
   - title: '🏗  Maintenance'
     labels:
-      - maintenance
-      - dependencies
+      - 🧹 maintenance
+      - ⬆️ ⬇️ dependencies
```

Now I'm being pedantic, but I feel like mutli-emojis should not have spaces between
```suggestion
      - ⬆️⬇️ dependencies
```